test/events.js

```diff
@@ -25,52 +25,6 @@ describe('events', function() {
       });
     };
   });
-
-  describe('changed', function() {
```

Comment:
AFAICT, there is no test remaining in this file after this change. Can we remove the entire file?